onboarding/time-management.md

```diff
@@ -0,0 +1,201 @@
+
+1. **See what you can do, and then**
+2. **Check out the resources you need to make more progress.** 
+
+This will help you avoid going too far into the available resources and give you a motivator.
+If you're working in a team, **check in with others** to stay focused on your goals. Team standups are a great place to start.
+
+---
+
+### Optimizing Your Calendar
+An effective calendar helps you stay organized, prioritize tasks, and balance work and personal time for increased productivity and well-being.
+
+#### Benefits of an effective Calendar:
+- **Ease of Mind**: Never forget an important event.
+- **Declutter Your Mind**: Less to keep track of mentally.
+- **Increase Confidence**: Tackle important tasks and schedule “me” time.
+- **Become Aware of Your Time**: See how you’re spending your time.
+- **Identify Priorities**: Notice what you’re over or under-prioritizing.
+- **Have More Quality, Focused Days**: Stay flexible while staying productive.
+
+#### Tips on how to make the best use of your Calendar:
+- **Schedule in Your WRAP**: Make time for your WRAP to ensure balance and wellness.
+- **Manage Self-Care**: Use your calendar to prioritize self-care. Feeling better helps you accomplish more with less stress. Avoid burnout by scheduling breaks.
+- **Plan Your Meals**: Schedule your meals for the day. Plan ahead, so you can buy any missing ingredients, make better food choices, and save mental energy for important tasks like learning new coding topics.
+- **Schedule Fun**: Include enjoyable activities throughout the week. Even 30 minutes dedicated to a hobby can help recharge you and bring joy.
+
+---
+
+### Tackling Procrastination
+
+Why do people procrastinate? Here are some common reasons:
+
+- Perfectionism/fear of negative evaluation
+- The need for time pressure
+- The issue seems overwhelming
+- Difficulty finding a starting point
+- A task is not appealing
+- Wanting a big chunk of time
+
+One key strategy for overcoming procrastination is to identify and challenge the unhelpful automatic thoughts that make tasks feel more overwhelming or unrealistic.
+
+**Challenging Unhelpful Automatic Thinking**
+
+Overcoming procrastination often involves challenging overly optimistic and unhelpful automatic thinking. These cognitive distortions can make tasks feel more daunting or unrealistic than they really are.
+
+
+### Challenging Cognitive Distortions:
+
+1. **Is this thought really true? What evidence or exceptions are there?**
+2. **Is this thought helpful?**
+3. **How else can I think about this?**
+4. **What would I say to a friend in this situation?**
+5. **Am I making assumptions or generalizations?**
+
+### Example Situation:
+
+**Situation**: Working on a difficult ticket  
+
+**Automatic thoughts**:  
+- *"I have to do all of this today. I must do it perfectly. If I don’t finish, my boss will be mad.
+If my work isn’t perfect and timely, my boss will be mad and I’ll be fired.
+"*
```

Comment:
It is unclear if the quoted lines are one or two statements. If a singular statement, changes should look like: 

```suggestion

**Automatic thoughts**:  "I have to do all of this today. I must do it perfectly. If I don’t finish, my boss will be mad. If my work isn’t perfect and timely, my boss will be mad and I’ll be fired."
```

If several statements, changes should look like: 
```suggestion

**Automatic thoughts**:  
- "I have to do all of this today." 
- "I must do it perfectly."
- "If I don’t finish, my boss will be mad."
- "If my work isn’t perfect and timely, my boss will be mad and I’ll be fired."
```

note in both suggestions, the quoted items are not bolded

Reply:
Hi @mai-repo can you leave a comment or even a commit hash clarifying where this has been addressed? Please do this before resolving each change request